### PR TITLE
ACS-5993 Classpath support

### PR DESCRIPTION
**Pull request by tpage-alfresco**
This is needed for certain rules (e.g. MissingOverride) to run successfully.

### CODE REVIEW

**Comment by tpage-alfresco** [comment]
There is an example PR here: https://github.com/Alfresco/alfresco-community-repo/pull/2301

File: README.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ steps:
       pmd-ruleset-repo: "Alfresco/pmd-ruleset" # The GitHub repository containing the PMD ruleset (by default https://github.com/Alfresco/pmd-ruleset/).
       pmd-ruleset-ref: "master" # The git reference (e.g. branch name, tag name or commit id) for the ruleset project.
       pmd-ruleset-path: "pmd-ruleset.xml" # The path to the PMD ruleset file from the root of the ruleset project. Optionally other paths to local rulesets can be appended to this separated by commas.
+      classpath-enable: "true" # Whether to set the classpath before the scan (used by certain rules - for example MissingOverride). This assumes the project uses maven.
```
Comment (tpage-alfresco) [inline README.md:26]: By defaulting this to true then this change is not backwards compatible. Consequently I think the release will be a major version bump.

**Review by Pawel-608 — APPROVED** [review]
LGTM

**Review by mpichura — APPROVED** [review]
LGTM

**Pull request merged**